codenotary.repo

```diff
@@ -0,0 +1,6 @@
+[codenotary]
```

Comment:
Does it exist as some remote file which can be downloaded? I guess it will be better than creating repo file by ourself.

Reply:
Good point, fixed